src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -257,7 +268,12 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                         access = dockerAccessFactory.createDockerAccess(dockerAccessContext);
                     }
                     ServiceHub serviceHub = serviceHubFactory.createServiceHub(project, session, access, log, logSpecFactory);
-                    executeInternal(serviceHub);
+                    try {
+                        executeInternal(serviceHub);
+                    } catch (Exception ex) {
```

Comment:
Is this additional try-catch block necessary?

Reply:
removed